src/config/index.js

```diff
@@ -1,41 +0,0 @@
-import firebase from "firebase/app";
```

Comment:
I think this will delete the file in this repo. Thought it was only ignoring. Do whatever @thisuraseniya asked you to do.

Reply:
Sorry. I will add the file and gitignore it.